extensions/amp-analytics/0.1/variables.js

```diff
@@ -220,7 +220,7 @@ export class VariableService {
         // Expand string values further.
         p = this.expandTemplate(raw,
             new ExpansionOptions(options.vars, options.iterations - 1,
-                options.noEncode));
+                true /* noEncode */));
```

Comment:
Found what I was looking for at https://github.com/lannka/amphtml/blob/a23f663af37c8283e10650df037f5e11841cae38/extensions/amp-analytics/0.1/variables.js#L236. It shows that this (now unencoded) recursive template would eventually be encoded.